src/main/java/com/minecolonies/coremod/entity/ai/mobs/util/MobEventsUtils.java

```diff
@@ -35,10 +35,24 @@
     private static final int    PREFERRED_MAX_CHIEFS         = 2;
     private static final int    MIN_CITIZENS_FOR_RAID        = 5;
     private static final int    NUMBER_OF_CITIZENS_NEEDED    = 5;
+
+    /**
+     * Different horde ids and their sizes.
+     */
+    private static final int SMALL_HORDE_MESSAGE_ID = 1;
+    private static final int MEDIUM_HORDE_MESSAGE_ID = 2;
+    private static final int BIG_HORDE_MESSAGE_ID = 3;
+    private static final int HUGE_HORDE_MESSAGE_ID = 4;
```

Comment:
no size for huge?

Reply:
huge is everything which is bigger than big =D